### PR TITLE
Project Listing | Implement basic project listing with Material UI table #4672

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/4672

### CODE REVIEW

File: moped-editor/src/views/projects/ProjectsTable.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                        </TableHead>
+                        <TableBody>
+                            {projects.map(project => (
+                                <TableRow hover key={project.project_id}>
+                                    <TableCell align="center">
+                                        <RouterLink to={`/moped/project/${project.project_id}`}>
+                                            <EditIcon color="primary" />
+                                        </RouterLink>
+                                    </TableCell>
+                                    <TableCell style={{"text-transform": "capitalize"}}><Link href={"/moped/project/"+project.project_id}>{project.project_name}</Link></TableCell>
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/ProjectsTable.js:47]: maybe this styling could be defined in the makeStyles object? No big deal I guess.

File: moped-editor/src/views/projects/ProjectsListView.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+const StaffListView = () => {
+    const classes = useStyles();
+    const { data, loading, error } = useQuery(projectsQuery.gql);
+
+    if (error) {
+        console.log(error);
+    }
+
+    if(data) {
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/ProjectsListView.js:96]: I noticed some formatting here with missing spaces. ESLint and Prettier config files at the root should work if you enable it in your code editor. Again, not a big deal.

**Review by mateoclarke — APPROVED** [review]
sonic

**Review by mddilley — APPROVED** [review]
Great stuff, Sergio - happy to see the GraphQL abstract in another component! 🚀

**Pull request merged**